api/tests/Feature/TeamsTest.php

```diff
@@ -216,43 +244,120 @@ public function testTeamUpdateMutation(): void
           ],
-          'contactEmail' => 'test@test.com',
+          'contactEmail' => 'newContactEmail@test.com',
           'departments' => [
-            "sync" => [$departmentId],
+            'sync' => [$departmentId],
           ],
         ]
       ]
     )->assertJson([
       'data' => [
         'updateTeam' => [
-          'name' => 'team one',
+          'name'=> 'new_team_name',
           'displayName' => [
-            'en' => 'en',
-            'fr' => 'fr',
+            'en' => 'New Team Name EN',
+            'fr' => 'New Team Name FR',
           ],
           'description' => [
-            'en' => 'en',
-            'fr' => 'fr',
+            'en' => 'New Team Description EN',
+            'fr' => 'New Team Description FR',
           ],
-          'contactEmail' => 'test@test.com',
+          'contactEmail' => 'newContactEmail@test.com',
           'departments' => [
-            [
-              'id' => $departmentId,
-            ],
+            ['id' => $departmentId],
           ],
         ]
       ]
     ]);
   }
+
+  public function testTeamDeleteMutation(): void
+  {
+    // Detach departments before deleting team
+    $this->team3->departments()->detach();
+
+    // Assert team update successful across all input fields
+    $this->actingAs($this->admin, "api")->graphQL(
+      /** @lang GraphQL */
+      '
+      mutation deleteTeam($id: UUID!) {
+          deleteTeam(id: $id) {
+            id
+          }
+      }
+      ',
+      [ 'id' => $this->toBeDeletedUUID, ]
+    )->assertJson([
+      'data' => [
+        'deleteTeam' => [
+          'id' => $this->toBeDeletedUUID,
+        ],
+      ]
+    ]);
+  }
+
+  public function testViewAnyTeamMembers(): void
+  {
+    // Attach all users to team one
+    $this->admin->attachRole("pool_operator", $this->team1);
+    $this->poolOperator1->attachRole("pool_operator", $this->team1);
+    $this->poolOperator2->attachRole("pool_operator", $this->team1);
+
+    // Assert all teams query contains expected results
+    $query = $this->actingAs($this->admin, "api")
+      ->graphQL(/** @lang Graphql */ '
+          query team($id: UUID!) {
+            team(id: $id) {
+                id
+                roleAssignments {
+                  id
+                  user {
+                    id
+                  }
+                }
+            }
+          }
+    ', [ 'id' => $this->team1->id ]);
+
+    $data = $query->original['data'];
+
+    // assert the teams returned is an array of exactly three items
+    $teamMembersCount = count($data['team']['roleAssignments']);
+    assertEquals($teamMembersCount, 3);
+
+    // assert every team member is present in the response
+    $query->assertJsonFragment(['id' => $this->admin->id]);
+    $query->assertJsonFragment(['id' => $this->poolOperator1->id]);
+    $query->assertJsonFragment(['id' => $this->poolOperator2->id]);
+  }
+
+  public function testViewTeamTeamMembersPolicy()
```

Comment:
After my other suggested changes, I think this test becomes unnecessary.

Reply:
Okay cool, removed in [ba0a1e6](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5899/commits/ba0a1e678c05c58f7c2398c9c86c468c66cf68e1)